src/spaceone/inventory/manager/workload/deployment_manager.py

```diff
@@ -180,11 +179,19 @@ def _get_pods(self, list_pods, list_replicasets, deployment_uid):
         pods_for_deployment = []
         for pod in list_pods:
             raw_pod = pod.to_dict()
+
+            if raw_pod.get("metadata").get("owner_references") is None:
+                continue
+
```

Comment:
Since `raw_replicaset.get("metadata")` is being repeated, it would be good to declare `metadata = raw_replicaset.get("metadata")` at the front of the code and avoid repetition :)